sdk/identity/azure-identity/src/main/java/com/azure/identity/DefaultAzureCredentialBuilder.java

```diff
@@ -34,6 +36,28 @@ public DefaultAzureCredentialBuilder authorityHost(String authorityHost) {
     }
 
 
+    /**
+     * Specifies the keep pass database path to read IntelliJ credentials on windows platform. This is required
+     * only on windows platform.
+     *
+     * <p>This path can be located in the IntelliJ IDE.
+     * Windows: File -&gt; Settings -&gt; Appearance &amp; Behavior -&gt; System Settings -&gt; Passwords </p>
+     *
+     * @param databasePath the path to the keep pass database.
+     * @throws IllegalArgumentException if {@code databasePath is either not specified or is empty}
+     * @return An updated instance of this builder with the keep pass database path set as specified.
+     */
+    public DefaultAzureCredentialBuilder windowsKeepPassDatabasePath(String databasePath) {
```

Comment:
Is there ever a possibility that there might be multiple databases on a users system that might need reading?

Reply:
IDE allows users to configure only one database at a time.
This will only be possible, if IDE allows users to configure multiple databases.